{"version":3,"sources":["../../src/api/messages.js"],"names":["defineMessages","messages","appActivityDeleteErrorMessage","commentCreateErrorMessage","commentCreateConflictMessage","commentDeleteErrorMessage","commentUpdateErrorMessage","taskActionErrorTitle","taskApproveErrorMessage","taskCompleteErrorMessage","taskRejectErrorMessage","taskDeleteErrorMessage","taskCreateErrorMessage","taskGroupExceedsLimitWarningMessage"],"mappings":"AAAA;;;;;AAMA,SAASA,cAAT,QAA+B,YAA/B;AAEA,IAAMC,QAAQ,GAAGD,cAAc,CAAC;AAC5BE,EAAAA,6BAA6B;AAAA;AAAA;AAAA,GADD;AAM5BC,EAAAA,yBAAyB;AAAA;AAAA;AAAA,GANG;AAW5BC,EAAAA,4BAA4B;AAAA;AAAA;AAAA,GAXA;AAgB5BC,EAAAA,yBAAyB;AAAA;AAAA;AAAA,GAhBG;AAqB5BC,EAAAA,yBAAyB;AAAA;AAAA;AAAA,GArBG;AA0B5BC,EAAAA,oBAAoB;AAAA;AAAA;AAAA,GA1BQ;AA+B5BC,EAAAA,uBAAuB;AAAA;AAAA;AAAA,GA/BK;AAoC5BC,EAAAA,wBAAwB;AAAA;AAAA;AAAA,GApCI;AAyC5BC,EAAAA,sBAAsB;AAAA;AAAA;AAAA,GAzCM;AA8C5BC,EAAAA,sBAAsB;AAAA;AAAA;AAAA,GA9CM;AAmD5BC,EAAAA,sBAAsB;AAAA;AAAA;AAAA,GAnDM;AAwD5BC,EAAAA,mCAAmC;AAAA;AAAA;AAAA;AAxDP,CAAD,CAA/B;AAiEA,eAAeZ,QAAf","sourcesContent":["/**\n * @flow\n * @file i18n messages\n * @author Box\n */\n\nimport { defineMessages } from 'react-intl';\n\nconst messages = defineMessages({\n    appActivityDeleteErrorMessage: {\n        id: 'be.api.appActivityDeleteErrorMessage',\n        description: 'Error message when an app activity deletion fails',\n        defaultMessage: 'There was an error deleting this item.',\n    },\n    commentCreateErrorMessage: {\n        id: 'be.api.commentCreateErrorMessage',\n        description: 'Error message when a comment creation fails',\n        defaultMessage: 'There was an error creating this comment.',\n    },\n    commentCreateConflictMessage: {\n        id: 'be.api.commentCreateConflictMessage',\n        description: 'Error message when a comment creation fails due to a conflict',\n        defaultMessage: 'This comment already exists.',\n    },\n    commentDeleteErrorMessage: {\n        id: 'be.api.commentDeleteErrorMessage',\n        description: 'Error message when a comment deletion fails',\n        defaultMessage: 'There was an error deleting this comment.',\n    },\n    commentUpdateErrorMessage: {\n        id: 'be.api.commentUpdateErrorMessage',\n        description: 'Error message when an annotation comment update fails',\n        defaultMessage: 'This comment could not be modified.',\n    },\n    taskActionErrorTitle: {\n        id: 'be.api.taskActionErrorTitle',\n        description: 'Title shown when an error occurs performing an action on a task',\n        defaultMessage: 'Error',\n    },\n    taskApproveErrorMessage: {\n        id: 'be.api.taskApproveErrorMessage',\n        description: 'Error message when approving a task fails',\n        defaultMessage: 'An error has occurred while approving this task. Please refresh the page and try again.',\n    },\n    taskCompleteErrorMessage: {\n        id: 'be.api.taskCompleteErrorMessage',\n        description: 'Error message when completing a task fails',\n        defaultMessage: 'An error has occurred while completing this task. Please refresh the page and try again.',\n    },\n    taskRejectErrorMessage: {\n        id: 'be.api.taskRejectErrorMessage',\n        description: 'Error message when rejecting a task fails',\n        defaultMessage: 'An error has occurred while rejecting this task. Please refresh the page and try again.',\n    },\n    taskDeleteErrorMessage: {\n        id: 'be.api.taskDeleteErrorMessage',\n        description: 'Error message when a task deletion fails',\n        defaultMessage: 'There was an error while deleting this task. Please refresh the page and try again.',\n    },\n    taskCreateErrorMessage: {\n        id: 'be.api.taskCreateErrorMessage',\n        description: 'Error message when a task creation fails',\n        defaultMessage: 'An error occurred while creating this task. Please try again.',\n    },\n    taskGroupExceedsLimitWarningMessage: {\n        id: 'be.api.taskGroupExceedsWarningTitle',\n        description:\n            'Warning message showing that, while the task was being created, the group assignee limit was exceeded by specified amount',\n        defaultMessage:\n            'One or more groups can not receive this task as a group size cannot exceed the limit of {max} assignees per group.',\n    },\n});\n\nexport default messages;\n"],"file":"messages.js"}